### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Supermegadex** [Supermegadex/libassist-client#1]
As a web developer, I should be doing this from the start. But I decided not to. This is going to come back to hurt me later, so if anyone is interested in helping me make this app ARIA-compliant, please make changes and send PRs!
